Configure

```diff
@@ -2673,6 +2673,12 @@ make)
 	gmake)
 	echo "I can't find make or gmake, and my life depends on it." >&4
 	echo "Go find a public domain implementation or fix your PATH setting!" >&4
+
+	test -f /etc/debian_version && echo "Run \"apt install build-essential\" to install make and friends" >&4
```

Comment:
`test -e` would be true for directories as well, so using `-f` here to limit the detection to files.